docs/project.md

```diff
@@ -3,3 +3,12 @@ sidebar_position: 3
 ---
 
 # Project
+
+# Rules
+1. It is not allowed to use processors (eg Broadcom BCM2711, Broadcom BCM2712), you can only use microprocessors. Examples of microprocessors: RP2040, ATmega328, nRF52, PIC.
+2. It is not allowed to copy both the code and the electronic schemes from the internet.
+3. For the hardware part, both the breadboard and printed pcbs or test cables can be used.
+4. Help from external sources is not allowed, the student can be asked by the coordinator about implementation details during the development period.
+5. The student can ask questions to the coordinator during the development period.
+6. Presentation of milestones is mandatory.
+7. Projects must be accompanied by documentation, this should include project functionality, hardware development and software development.
```

Comment:
This needs to be detailed.

Reply:
For the moment can detail from this:
https://ocw.cs.pub.ro/courses/pm/pm/prj2021/template
I'll do one template in english before start of semester.